### PR TITLE
chore: add copyright bump task to release train issue template

### PR DESCRIPTION
**Pull request by sebastian-alfers**

### CODE REVIEW

File: akka-docs/release-train-issue-template.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ Key links:
 - [ ] Check that open PRs and issues assigned to the milestone are reasonable
 - [ ] If PRs were merged after EU midnight, trigger the [native-image tests](https://github.com/akka/akka/actions/workflows/native-image-tests.yml) and see that they are green.
 - [ ] Update the version and change date in the LICENSE file.
+- [ ] Allign the copyright year in all files.
```
Comment (johanandren) [inline akka-docs/release-train-issue-template.md:22]:
```suggestion
- [ ] Align the copyright year in all files.
```

**Review by johanandren — APPROVED** [review]

File: akka-docs/release-train-issue-template.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ Key links:
 - [ ] Check that open PRs and issues assigned to the milestone are reasonable
 - [ ] If PRs were merged after EU midnight, trigger the [native-image tests](https://github.com/akka/akka/actions/workflows/native-image-tests.yml) and see that they are green.
 - [ ] Update the version and change date in the LICENSE file.
+- [ ] Align the copyright year in all files.
```
[the remark this answers is not in the document]
Comment (ennru) [reply akka-docs/release-train-issue-template.md:22]: So normally just check the box.

**Review by ennru — APPROVED** [review]
LGTM.

File: akka-docs/release-train-issue-template.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (patriknw) [reply akka-docs/release-train-issue-template.md:22]: eh, that has no purpose
If we want to become better at updating the year we should just have a calendar task once per year and do it. Not related to releases.

**Pull request closed without merging**